tests/test_api.py

```diff
@@ -773,9 +773,9 @@ def test_generate_manifest_file_based_annotations(
 
         # make sure Filename, entityId, and component get filled with correct value
         assert google_sheet_df["Filename"].to_list() == [
-            "TestDataset-Annotations-v3/Sample_A.txt",
-            "TestDataset-Annotations-v3/Sample_B.txt",
-            "TestDataset-Annotations-v3/Sample_C.txt",
+            "schematic - main/TestDataset-Annotations-v3/Sample_A.txt",
```

Comment:
The code ends up calling to `getFilesInStorageDataset` where the file names are retrieved. In that function we are including the project within the file name.